addons/csi-vsphere/1-vsphere-csi-driver.yaml

```diff
@@ -167,21 +167,8 @@ roleRef:
 apiVersion: v1
 data:
   "csi-migration": "{{ .CSIMigration }}"
-  "csi-auth-check": "true"
-  "online-volume-extend": "true"
   "trigger-csi-fullsync": "false"
-  "async-query-volume": "true"
-  "block-volume-snapshot": "true"
-  "csi-windows-support": "false"
-  "use-csinode-id": "true"
-  "list-volumes": "true"
   "pv-to-backingdiskobjectid-mapping": "false"
-  "cnsmgr-suspend-create-volume": "true"
-  "topology-preferential-datastores": "true"
-  "max-pvscsi-targets-per-vm": "true"
-  "multi-vcenter-csi-topology": "true"
-  "csi-internal-generated-cluster-id": "true"
-  "listview-tasks": "false"
```

Comment:
Why did we drop all these options?

Reply:
These were all removed in https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/2781 between 3.1 and 3.2 and cannot be set to `false` anymore. `listview-tasks` was upstream already `true`, not sure why we forgot to change (?) our value for that feature gate.

Reply:
"staying on 2.7" is impossible. KubeOne is already on 3.1.2. This PR here updates KubeOne from 3.1.2 to 3.2.0 and 3.2.0 removes the feature gates.

Reply:
Ah, then I indeed misunderstood your comment. Thanks for clarifying this!